### ISSUE TITLE
Annotating a Poetry Reading

### ISSUE THREAD

**Issue by saracarl** [IIIF/cookbook-recipes#103]
## Annotating a Poetry Reading

(Provide link to `index.md` of the issue, if available. To be filled in after issue is created - you need the issue number!)

## Use case  
While an audio file of a poetry performance may be divided into a track for each poem, scholars may wish to use annotations to indicate aspects of the performance of a particular poem.  

Examples of the kinds of information to be annotated may include the following:
* structural information (introduction, title, stanzas)
* points where the performer takes a breath
* repeated phrases

**Comment by saracarl** [comment]
We should add this to 
https://github.com/BrumfieldLabs/cookbook-recipes/blob/master/recipe/index.md#other-kinds-of-annotations
when we're done.

**Comment by saracarl** [comment]
To Test:  If we have web annotations in a separate file, can we just include them in the recipe's directory?  Will they get built?   How and where do we references them (outside of the manifest?)

**Comment by glenrobson** [comment]
Yes the annotations file should be in the same directory as the manifest. Regarding referencing them please give me a shout when this is ready so we can try a few options.

**Comment by glenrobson** [comment]
Done in: https://github.com/IIIF/cookbook-recipes/pull/130
